linkerd-network-validator.yaml

```diff
@@ -45,4 +45,4 @@ test:
   pipeline:
     - runs: |
         /usr/lib/linkerd/linkerd2-network-validator --version | grep ${{package.version}}
-        /usr/lib/linkerd/linkerd2-network-validator --help
+        # /usr/lib/linkerd/linkerd2-network-validator -h
```

Comment:
Why are we commenting this out?

Reply:
> Batuhan Apaydin
  12:13 PM
they have to remove the help command I don't know I tried to remove it and it worked :melting_face:

The issue is in upstream they have updated their `clap` dependency from `v3` to `v4` which is a [breaking change](https://github.com/clap-rs/clap/blob/master/CHANGELOG.md#400---2022-09-28). upstream binary is also failing running the `--help` command.